### DISCUSSION TITLE
ton+eth increase efficiency

### DISCUSSION THREAD

**Discussion by karmagkz** [Lolliedieb/lolMiner-releases#945] [Ideas]
1,42v in the current situation when starting ton + eth mining with 2 .bat
ton efficiency drops by almost 3 times
eth efficiency drops by 2 times

is it possible to distribute the load between gpu and memory for two different coins to increase efficiency?

**Comment by darius163** [comment]
a very necessary dual mining. Try to make it as profitable as possible, please

**Comment by DiDragon2022** [comment]
Please try to make such a dual ton+eth, it will be in great demand... and people will be even more interested in your lolMiner..

**Reply by eaamgh** [reply]
You should now try 1.43 and DUALTON mode. While some are experiencing issues with this, I've got it running on 6900xt's and a non-LHR 3070. Takes more power, but no impact on ETH mining at all. Just have to take care about new power requirements...
